pkg/parse/fs.go

```diff
@@ -26,8 +27,8 @@ type fsFileStream struct {
 	filename string
 }
 
-func newFSFileStream(filename string, fs afero.Fs) (*fsFileStream, error) {
-	f, err := fs.Open(filename)
+func newFSFileStream(filename string, fs afero.Fs) (*fsFileStream, *mod.Module, error) {
```

Comment:
```suggestion
func newFSFileStream(filename string, fs mod.Fs) (*fsFileStream, *mod.Module, error) {
```

Reply:
Actually I think it might be better to create a new Fs interface inside of the mod package, that embeds the afero.Fs interface